packages/dx-react-grid/docs/guides/data-accessors.md

```diff
@@ -74,8 +74,6 @@ The following demo shows this approach in action:
 
 .embedded-demo(data-accessors/custom-data-accessors)
 
-Note that the Grid's `getCellValue` property has a higher priority than the column's property.
```

Comment:
We have changed the priorities. Why do we remove the note completely instead of changing it?

Reply:
We haven't such note in other places. We don't need to leave it here for now.